src/utils/__tests__/parseHelpers.test.js

```diff
@@ -0,0 +1,107 @@
+const parseHelpers = require('../preview/parseHelpers');
+const {
+  expectedConfigOutput,
+  missingConfigOutput,
+  configContentMissingFields,
+  configContent,
+  installPlansInput,
+  baseFiles,
+  dashboardFiles,
+  dashboardContent,
+  expectedDashboardOutput,
+  installPlansOutput,
+  documentationInput,
+  documentationOutput,
+} = require('../mock_data/content');
+
+console.log(baseFiles);
+
+describe('Action: Parse helpers', () => {
```

Comment:
I think the `Action:` part here was only used for tests that relate to Github actions, so let's just name it after the file it's testing.
```suggestion
describe('parseHelpers', () => {
```